README.md

```diff
@@ -201,23 +299,27 @@ but method extraction is **already possible** with this proposal.\
 is not much wordier than\
 `const slice = arr&.slice; slice(1, 3);`
 
-**Extension getters and setters**
-(i.e., extending objects with new property getters or setters
-**without mutating** the object)
-may **also** be useful,
-and this proposal would be **forward compatible** with such a feature
-using the **same operator** `->` for **property-object binding**,
-in addition to this proposal’s **method binding**.
-Getter/setter binding could be added in a separate proposal
-using `{ get () {}, set () {} }` objects.
-For example, we could add an extension getter `allDivs`
-to a `document` object like so:
+**Extracting property accessors** (i.e., getters and setters)
+is not a goal of this proposal.
+Get/set accessors are **not like** methods. Methods are **values**.
+Accessors themselves are **not values**;
+they are functions that activate when getting or setting properties.
+Getters/setters have to be extracted using `Object.getOwnPropertyDescriptor`;
+they are not handled in a special way.
+This verbosity may be considered to be desirable [syntactic salt][]:
+it makes the developer’s intention (to extract getters/setters – and not methods)
+more explicit.
+
 ```js
-const allDivs = {
-	get () { return this.querySelectorAll('div'); }
-};
+const { get: $getSize } =
+  Object.getOwnPropertyDescriptor(
+    Set.prototype, 'size');
 
-document->allDivs;
+// The adversary’s code.
+delete Set; delete Function;
+
+// Our own trusted code, running later.
+new Set([0, 1, 2])->$getSize();
```

Comment:
FWIW this is also dependent on `Array.prototype[Symbol.iterator]` (and `%ArrayIteratorPrototype%.next`) but maybe it's ok for this example.

Reply:
Yeah, let’s leave it for now in this pull request. 
If you come up with a more robust example, I’d be happy to merge a separate pull request. Thank you again for reviewing. : )